Makefile

```diff
@@ -386,7 +387,41 @@ helm-chart: manifests kustomize helmify
 unit-test: ## Run unit tests for the helm chart
 	PATH="$(CURDIR)/scripts:$(PATH)" bats test/unit/
 
+##@ GKE
+
+.PHONY: create-gke
+create-gke: ## Create a new GKE cluster
+	$(TERRAFORM) -chdir=$(TF_GKE_DIR) init -upgrade
+	$(TERRAFORM) -chdir=$(TF_GKE_DIR) apply -auto-approve
+	$(GCLOUD) container clusters get-credentials $$($(TERRAFORM) -chdir=$(TF_GKE_DIR) output -raw kubernetes_cluster_name) \
+	--region $$($(TERRAFORM) -chdir=$(TF_GKE_DIR) output -raw region)
+
+# Currently only supports amd64
+.PHONY: ci-gar-build-push
+ci-gar-build-push: ## Build the operator image and push it to the GAR repository
+	@$(eval GCP_REGION := $(shell $(TERRAFORM) -chdir=$(TF_GKE_DIR) output -raw region))
+	@$(eval GCP_PROJ_ID := $(shell $(TERRAFORM) -chdir=$(TF_GKE_DIR) output -raw project_id))
+	@$(eval GCP_GAR_NAME := $(shell $(TERRAFORM) -chdir=$(TF_GKE_DIR) output -raw gar_name))
+	@$(eval IMAGE_TAG_BASE := $(GCP_REGION)-docker.pkg.dev/$(GCP_PROJ_ID)/$(GCP_GAR_NAME)/$(BIN_NAME))
```

Comment:
that's a log of TF calls, I wonder if we could capture the JSON output once, the use jq to extract the values that we want?

Something like:
```
terraform output -json | jq -re '.region.value'
```

Reply:
good idea, let me try that